Yoast/wordpress-seo
# Add loading indicator to MailChimp component

## Summary

This PR can be summarized in the following changelog entry:
- Add loading indicator to the mail chimp sign-up to provide feedback to the user when performing a request in the back-end.

## Relevant technical choices:
- Reused the loading indicator component from the yoast-components repo.

## Test instructions

This PR can be tested by following these steps:
1. Start the wizard and go to step 1
2. Fill in your email and sign-up. The page should grey out and show a loading indicator until you see a response.


Fixes https://github.com/Yoast/wordpress-seo/issues/5564
